addons/mail/static/src/discuss/core/common/channel_member_list.xml

```diff
@@ -27,10 +27,11 @@
     </t>
 
     <t t-name="discuss.channel_member">
-        <div class="o-discuss-ChannelMember d-flex align-items-center p-2 bg-view" t-att-class="{ 'cursor-pointer': canOpenChatWith(member) }" t-on-click.stop="() => this.openChatAvatar(member)">
+        <div class="o-discuss-ChannelMember d-flex align-items-center p-2 bg-view">
             <div class="bg-inherit o-discuss-ChannelMember-avatar position-relative d-flex ms-4 flex-shrink-0">
                 <img class="w-100 h-100 rounded o_object_fit_cover"
-                     t-att-src="member.persona.avatarUrl"/>
+                     t-att-src="member.persona.avatarUrl"
+                     t-att-class="{ 'cursor-pointer': canOpenChatWith(member) }" t-on-click.stop="(ev) => this.onClickAvatar(ev, member)"/>
```

Comment:
It's a better UX to make click on whole item rather than just the avatar, similarly to how some other messaging apps do